File: backend/models/stop.go
package models

type Stop struct {
	Uuid                 string `gorm:"primary_key" json:"uuid"`
	ClassifiedPeriodUuid string `json:"classifiedPeriodUuid"`
	ReasonId             int64  `json:"reasonId"`
	GoogleMapsDataUuid   string `json:"googleMapsDataUuid"`
	UserId               int64
}

File: core_backend/core_backend/Models/Stop.cs
using System.ComponentModel.DataAnnotations;

namespace core_backend.Models
{
    public class Stop
    {
        [Key]
        public string Uuid { get; set; }
        public string ClassifiedPeriodUuid { get; set; }
        public long ReasonId { get; set; }
        public string GoogleMapsDataUuid { get; set; }
        public string UserId { get; set; }
    }
}
